x/group/internal/orm/genesis.go

```diff
@@ -0,0 +1,19 @@
+package orm
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// TableExportable
+type TableExportable interface {
+	// Export stores all the values in the table in the passed
+	// ModelSlicePtr. If the table has an associated sequence, then its
+	// current value is returned, otherwise 0 is returned by default.
+	Export(sdk.KVStore, ModelSlicePtr) (uint64, error)
+
+	// Import clears the table and initializes it from the given data
+	// interface{}. data should be a slice of structs that implement
+	// PrimaryKeyed. The seqValue is optional and only
+	// used with tables that have an associated sequence.
+	Import(sdk.KVStore, interface{}, uint64) error
```

Comment:
can we update `data` type (the second argument) to:
```
data []PrimaryKeyed
```

Reply:
```suggestion
	Import(store sdk.KVStore, data interface{}, seqValue uint64) error
```

Reply:
Having `data` as `[]PrimaryKeyed` would mean that we would need to manually build the slice of PrimaryKeyed based on the concrete type slice before using it in `Import` which seems a bit cumbersome